apps/examples/testcase/le_tc/kernel/tc_libc_string.c

```diff
@@ -39,9 +39,9 @@
 
 #define BUFF_SIZE 5
 #define BUFF_SIZE_10 10
-#define BUFF_SIZE_12 12
 
 #define EBUSY_STR_SIZE (sizeof(EBUSY_STR))
+#define EFAULT_STR_SIZE (sizeof(EFAULT_STR))
```

Comment:
EFAULT_STR_SIZE is used just once. How about using directly, not macro?

Reply:
I agree.
I will change sizeof(EFAULT_STR) instead of macro